### ISSUE TITLE
As a CAS user, I want to see user information when approving prime admins

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#869]
#### Description:

When CAS users are approving prime admins, they currently only see operator information. They should also see information about the prime admin.

Wireframe: https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=413-48&mode=design&t=bzAh8Xan6cB0GTcm-0 

#### Acceptance Criteria:

Given I am an internal use
When I click into the detail page of a prime admin request
Then I should see the prime admin's information (doesn't have to be in an accordion like the wireframe; it's fine to use the current design) which includes: all the user information we can get from their Business BCeID
- First Name
- Last Name
- Email Address
- Phone Number
- Position Title
- BCeID Business name
- ~Etc. (Let's confirm with other developers to see if there're additional fields that we can pull from the Business BCeID, @BCerki)~
- ^ I think the BCeID business name covers the ETC. above - Dylan

#### Development Checklist:

- [x] Show the user information (can do this within the multistep form or make the accordions, whatever's easier, accordions not needed for MVP)
- [x] Make sure approve/deny buttons correctly impact either operator or prime admin
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking
